modules/ztp-ai-install-ocp-clusters-on-bare-metal.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+.Enabling Assisted Installer Service on Bare Metal
+
+The Assisted Installer Service (AIS) deploys {product-title} clusters. {rh-rhacm-first} ships with AIS, which is deployed when the MultiClusterHub Operator is enabled on the {rh-rhacm} hub cluster.
+
+For distributed units DUs, {rh-rhacm} supports single node OpenShift deployments. Single node OpenShift runs on a single bare-metal host. When the host is provisoned, the single node cluster acts as both supervisor and worker node. Shift Container Platform {product-title} on top of a single BMO For distributed units DUs, {rh-rhacm} supports single node OpenShift deployments. Single node OpenShift runs on a single bare-metal host. When the host is provisoned, the single node cluster acts as both supervisor and worker node.
+
+.Enable assisted installer service
+
+.Prerequisites
+
+. Install {product-title} {product-verstion} on a hub cluster.
+. Install {rh-acm} and create the MultiClusterHub resource.
+. Create PersistentVolume custom resources (CR) for database and file system storage.
+. Install openshift-install CLI for the desired version of {product-title}.
```

Comment:
This should be an unordered list (bullets), not numbers. * instead of .

Reply:
Updated accordingly.